HW3/hw3.py

```diff
@@ -0,0 +1,78 @@
+from flask import Flask, request, jsonify, render_template_string
+from faker import Faker
+import random
+from datetime import datetime, timedelta
+import pandas as pd
+import requests
+import json
+
+app = Flask(__name__)
+def random_birthday():
+    start= datetime.strptime('1964-01-01', '%Y-%m-%d')
+    end = datetime.strptime('2006-12-31', '%Y-%m-%d')
+    return start + timedelta(days=random.randint(0, int((end - start).days)))
+
+@app.route('/generate_users', methods=['GET'])
+def generate_students():
+    # count should be as input GET parameter
+    count = int(request.args.get('count', 10))
+    if not isinstance(count, int):
+        return jsonify({'error': 'Input is not int'}), 400
+
+    faker_instance = Faker("uk_UA")
+    df = pd.DataFrame()
+    # set limit as 1000
+    for i in range(min(count, 1000)):
```

Comment:
1000 is not default value this is max value, if user use more than 1000 you need to notify